src/content/docs/tutorial-create-alerts/improve-with-alerts.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: "Manage your alert quality"
+metaDescription: "How manage your alerts and notifications"
+---
+
+import AQMdash from 'images/alerts_screenshot-crop_AQM-dash.webp'
+
+import AQMflow from 'images/alerts_diagram_AQM-flow.webp'
+
+As time goes on, your number of alerts will grow. This can lead to problems for your organization if they're not managed correctly. Additionally, your alerts will give you crucial information you can use to improve your system, and if you don't take advantage of that information, you won't be using your alerts to their fullest potential. Below, we've provided information on managing the quality of your alerts to prevent things like alert fatigue, as well as how you can use alerts to gather data and drive positive impact to your organization.
+
+<img
+    title="AQM dashboard"
+    alt="A screenshot displaying a view of an AQM dashboard in New Relic"
+    src={AQMdash}
+/>
+
+## Manage alert quality
+
+Alert quality management (AQM) is a concept focused on reducing unneccessary alerts and helping you ensure the alerts you receive are the most relevant ones. AQM follows a simple flow of installing the dashboard, getting information to establish your baselines, gathering data over a period of time, and then making changes based on that data to improve your stack and drive positive impact for your organization.
+
+<img
+    title="AQM flow"
+    alt="A diagram displaying how to use AQM to improve your system"
+    src={AQMflow}
+/>
+
+To get started with AQM, you need to do the following:
+
+<CollapserGroup>
+    <Collapser
+        id="AQM-dash"
+        title="1. Install the AQM dashboard"
+    >
+1. [Go to the **Alert Quality Management** instant observability page](https://newrelic.com/instant-observability/alert-quality-management)
```

Comment:
I don't think this entire sentence should be the link; probably just the bolded part.

Reply:
Good call!